diff --git pym/cl_vars_share.py pym/cl_vars_share.py
index 4de30c0..0b254b2 100644
--- pym/cl_vars_share.py
+++ pym/cl_vars_share.py
@@ -254,9 +254,11 @@
                      "CLS":"Calculate Linux Scratch",
                      "CSS":"Calculate Scratch Server",
                      "CMC":"Calculate Media Center",
+                     "COBRA":"rasdark Desktop Linux",
+                     "TOSS":"TFOKB Security Server",
                      "Gentoo":"Gentoo"}
 
-    dictLinuxSubName = {"CLD":"KDE", "CLDX":"XFCE", "CLDG":"GNOME"}
+    dictLinuxSubName = {"CLD":"KDE", "CLDX":"XFCE", "CLDG":"GNOME", "COBRA":"Minimal"}
 
     dictNameSystem = {'CDS':'server',
                       'CLD':'desktop',
@@ -264,7 +266,9 @@
                       'CLDX':'desktop',
                       'CLS':'desktop',
                       'CMC':'desktop',
-                      'CSS':'server'}
+                      'CSS':'server',
+                      'COBRA':'desktop',
+                      'TOSS':'server'}
 
     # data object from LDAP
     _ldapUserObject = False
